app/Policies/UserPolicy.php

```diff
@@ -137,6 +137,10 @@ public function updateAnyPermission(User $user, User $target, Role $role = null)
             ]);
         }
 
+        if ($role->name == Role::TENANT) {
+            return $user->hasRole([Role::STAFF, Role::STUDENT_COUNCIL => Role::PRESIDENT]);
+        }
```

Comment:
Then, you should add the president to the view policy as well (if not there already)